src/components/Breacrumb.svelte

```diff
@@ -0,0 +1,27 @@
+<script>
+  import { blogTitle } from '../stores/blogTitle';
+  import { stores } from '@sapper/app';
+
+  const { page } = stores();
+
+  export let segment;
+
+	const segmentData = {
+		profile: "プロフィール",
+		blog: "ブログ",
+		created: "製作",
+		contact: "お問い合わせ"
+	}
+</script>
+
+<header class="mb-4">
```

Comment:
```suggestion
<div class="mb-4">
```
`<h>`タグがないので`<header>`でなくて良い